tutorial.md

```diff
@@ -7,13 +7,24 @@
 
 # Customize a data warehouse With BigQuery Solution
 
-Learn how to build and deploy your own proof of concept solution based on the deployed [Data warehouse with BigQuery](https://console.cloud.google.com/products/solutions/details/data-warehouse) Jump Start Solution (JSS) and deploy it. You can customize the Jump Start Solution (JSS) deployment by creating a copy of the source code. You can modify the infrastructure and application code as needed and redeploy the solution with the changes.
+Learn how to build and deploy your own proof of concept solution based on the deployed [Data warehouse with BigQuery](https://console.cloud.google.com/products/solutions/details/data-warehouse) Jump Start Solution and deploy it. You can customize the Jump Start Solution deployment by creating a copy of the source code. You can modify the infrastructure and application code as needed and redeploy the solution with the changes.
 
 To avoid conflicts, only one user should modify and deploy a solution in a single GCP project.
 
 ## Open cloned repository as workspace
 
-Open the directory where the repository is cloned as a workspace in the editor.
+Open the directory where the repository is cloned as a workspace in the editor, follow the steps based on whether you are using the Cloud Shell Editor in Preview Mode or Legacy Mode.
+
+---
+**Cloud Shell Editor in Preview Mode**
```

Comment:
Is it preview mode? Or just New Cloud Shell Editor?

Reply:
Updated to "New Cloud Shell Editor"